autogluon/utils/tabular/ml/trainer/abstract_trainer.py

```diff
@@ -826,6 +829,38 @@ def _get_dummy_stacker(self, level, use_orig_features=True):
         )
         return dummy_stacker
 
+    # TODO: Add option for raw=True, which uses black-box method to get importances for original features instead of whatever features the model uses.
+    #  This is non-trivial to implement for multi-layer stacking ensembles on the OOF data.
+    def get_feature_importance(self, model, X=None, y=None):
```

Comment:
For presenting importances to user, trainer should eventually have a separate feature_importance() method that automatically uses the best model, and even if it's a stacker, only considers the original features for importance scoring.

Reply:
Agreed